content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -23,7 +23,7 @@ In addition to the out of the box (OOTB) [default Agent and detection rules][7],
 
 To prevent users with the [Datadog Standard out-of-the-box role][11] from creating, updating, disabling, and deploying a changed rule using remote configuration:
```

Comment:
Maybe this should change too? The `security_monitoring_cws_agent_rules_actions` permission is only about whether a customer could turn on and configure the CWS protection feature. Users with the standard role are able to create/update a custom rule by default, as long as the operation does not change the protection settings on the rule.